app/public/views/userView/data/pagesCellsSpecials.js

```diff
@@ -106,10 +106,14 @@ pagesCellsSpecials[PN.runsPerPeriod] = {
     }, item.fill_number, h('.external-link-15-blue')),
 };
 
+const linerUnderVerifier = '--------';
 pagesCellsSpecials[PN.flags] = {
     time_start: (item) => dateFormatter(item.time_start),
     time_end: (item) => dateFormatter(item.time_end),
-    ver_time: (item) => item.ver_time.isEmpty
+    by: (item) => ! item.by?.length
+        ? 'unverified'
+        : item.by.map((verifier) => h('', verifier, h('.skinny', linerUnderVerifier))),
+    ver_time: (item) => ! item.ver_time?.length
```

Comment:
Why ver_time becomes ver_time.length?

Reply:
I moved question mark to upper line

Reply:
My bad I did read too quickly, it was better before